salt/search/init.sls

```diff
@@ -23,6 +23,17 @@ search-repository:
         - require:
             - builder: search-repository
 
+{% if pillar.elife.env in ['dev', 'ci'] %}
+search-queue-create:
+    cmd.run:
+        - name: aws sqs create-queue --region=us-east-1 --queue-name=search--{{ pillar.elife.env }} --endpoint=http://localhost:4100
+        - cwd: /home/{{ pillar.elife.deploy_user.username }}
```

Comment:
I thought that was found with the `$HOME` environment variable, not with the current working directory?